cpt/builds_generator.py

```diff
@@ -6,11 +6,11 @@
 from conans.model.version import Version
 from cpt.tools import split_colon_env, transform_list_options_to_dict
 
-default_gcc_versions = ["4.9", "5", "6", "7"]
-default_clang_versions = ["3.8", "3.9", "4.0"]
-default_visual_versions = ["10", "12", "14"]
+default_gcc_versions = ["4.9", "5", "6", "7", "8"]
+default_clang_versions = ["3.8", "3.9", "4.0", "5.0", "6.0", "7.0"]
+default_visual_versions = ["14", "15"]
 default_visual_runtimes = ["MT", "MD", "MTd", "MDd"]
-default_apple_clang_versions = ["7.3", "8.0", "8.1"]
+default_apple_clang_versions = ["9.0", "9.1", "10.0"]
```

Comment:
sure about excluding apple clang 8? our packages in conan-center are supporting it.

Reply:
It doesn't really matter, almost no-one is running with the default configurations but specifying which ones have to be built. Our "policy" or goal is to build the two latest versions for apple-clang.